downstream/modules/platform/proc-controller-getting-started-edit-project.adoc

```diff
@@ -0,0 +1,31 @@
+[id="controller-getting-started-edit-project"]
+
+= Editing a project
+
+As part of the initial setup you can leave the default *Demo Project* as it is. 
+You can edit it later.
+
+.Procedure
+. Open the project to edit it by using one of these methods:
+** Go to the *Details* tab. 
+Click btn:[Edit].
+** Go to the *Projects* list view. 
+Click btn:[Edit} next to the project name and edit the appropriate details.
+. Save your changes
+
+
+If you want to fetch the project for the latest changes, you can manually start an SCM sync for this project:
```

Comment:
Have you said somewhere what an SCM sync is?

Reply:
This is defined as "source code management" in the `controller-projects` assembly.